File: Herança-Subclasses-de-Personagem/Item.py
class Item:
    def __init__(self, nome, bonus):
        self.nome = nome

        if bonus >= 0:
            self.bonus = bonus
        else:
            self.bonus = 0

    # Getters
    def getNomeItem(self):
        return self.nome

    def getBonusItem(self):
        return self.bonus

    # Setters
    def setNomeItem(self, nome):
        self.nome = nome

    def setBonusItem(self, bonus):
        if bonus >= 0:
            self.bonus = bonus
        else:
            print("\nInsira um valor válido para o bônus")

    # Métodos
    def descricao(self):
        print(f"{self.nome} (+{self.bonus})")

File: Item.java
public class Item{
    private String nome;
    private int bonus;

    public Item(String nome, int bonus){
        this.nome = nome;
        if(bonus >=0){
            this.bonus = bonus;
        }
    }

    //Getters
    public String getNomeItem() {
        return nome;
    }

    public int getBonusItem() {
        return bonus;
    }


    //Setters
    public void setNomeItem(String nome){
        this.nome = nome;
    }

    public void setBonusItem(int bonus){
        if(bonus >= 0){
            this.bonus = bonus;
        } else{
            System.out.printf("%nInsira um valor válido para o bônus%n");
        }
    }
    
    //Metodos
    public void descricao(){
        System.out.printf("%s (+%d)%n", nome, bonus);
    }

}